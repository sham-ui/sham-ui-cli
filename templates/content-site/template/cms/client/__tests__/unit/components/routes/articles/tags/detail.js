import * as directives from 'sham-ui-directives';
// eslint-disable-next-line max-len
import RoutesArticlesTagsDetail  from '../../../../../../src/components/routes/articles/tags/detail.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( RoutesArticlesTagsDetail, {}, {
        directives
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
