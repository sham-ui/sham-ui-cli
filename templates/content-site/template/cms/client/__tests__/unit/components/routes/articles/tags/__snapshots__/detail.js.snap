// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "item": Object {},
    "onUpdateSuccess": [Function],
  },
  "html": "
<h6>Tag: null</h6>
<form class=\\"form-layout\\">
  <!--0-->
  <!--0-->
  <!--1-->
  <div class=\\"form-group\\">
    <label>Name:</label>
    <input name=\\"name\\" type=\\"text\\" autofocus=\\"\\">
  </div>
  <!--0-->
  <!--2-->
  <button type=\\"submit\\">
    <!--3-->Update
    <!--4-->
  </button>
  <!--5-->
</form>
<!--0-->
<!--1-->
<!--0-->
<!--0-->
",
}
`;
